### ISSUE TITLE
Profile more metrics

### ISSUE THREAD

**Issue by lpefferkorn** [lpefferkorn/cfe-profiler#7]
Trying to add more metrics for each profiled bundle, as the number of calls to:
- file reads and writes
- execresult()
- filestat()
- ?

**Comment by TheDreamer** [comment]
Yes....I've been trying to figure out why my cf-agent runs since July 29th have been taking longer to complete and have become very write intensive.

Was fairly certain which bundle it was spending most of its time in, and have confirmed it now.  Though was surprising to see other time consuming bundles.
